base/Angle.hpp

```diff
@@ -106,6 +107,20 @@ class Angle
         return result;
     }
 
+    /** Use this method to get the minimum acceptable angle
```

Comment:
> The documentation is wrong

The documentation is indeed ambiguous. I would suggest either "Returns the minimum Angle" or "Returns the minimum angle possible after normalization"

(I guess that Max would need to be changed accordingly)